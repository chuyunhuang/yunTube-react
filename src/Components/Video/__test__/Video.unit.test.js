import React from 'react';
import { shallow } from 'enzyme';
import { Video } from '../Video';

describe('Video', () => {
  test('renders Video component correctly with an id', () => {
    const wrapper = shallow(<Video id="testing" />)
    expect(wrapper).toMatchSnapshot();
  });

  test('render null if an id is not given to the Video component', () => {
    const wrapper = shallow(<Video />)
    expect(wrapper).toMatchSnapshot();
  });
});