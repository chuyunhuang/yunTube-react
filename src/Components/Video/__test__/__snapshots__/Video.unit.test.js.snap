// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video render null if an id is not given to the Video component 1`] = `""`;

exports[`Video renders Video component correctly with an id 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/testing"
      title="video"
    />
  </div>
</div>
`;
